backup/Duke.java

```diff
@@ -0,0 +1,116 @@
+import java.io.*;
```

Comment:
One point that can be improved here is not by importing everything from a library. Consider this class Something which is included both in the java.io library and the java.util library. If you intended to use the class from the util library by calling Something something = new Something(); the compiler might accept it as the class from io instead. By importing individual classes, we do not have this problem. Keep it up!

Reply:
Thanks for the comments! I will take note to keep to coding standards :)